src/rustdoc.md

```diff
@@ -44,8 +44,8 @@ does is call the `main()` that's in this crate's `lib.rs`, though.)
 * Run `./x setup tools` before getting started. This will configure `x`
   with nice settings for developing rustdoc and other tools, including
   downloading a copy of rustc rather than building it.
-* Use `./x check src/tools/rustdoc` to quickly check for compile errors.
-* Use `./x build` to make a usable
+* Use `./x check rustdoc` to quickly check for compile errors.
+* Use `./x build library rustdoc` to make a usable
```

Comment:
I always use `./x.py build --stage 1 -j8` for it to work for me, even now. Strange...

Reply:
I see, you're also rebuilding rustc before building rustdoc. I however build stage2 rustdoc with a downloaded rustc (if unchanged) to save compute power & time. This is the default for profile `tools`.

Reply:
So what I'm saying is that anyone who also builds stage2 rustdoc now needs to build the sysroot explicitly since `./x b library rustdoc` is equivalent to `./x b --stage 2 library rustdoc` if you're using the default bootstrap configuration.

Reply:
And this document assumes that you're building stage2 rustdoc.

Reply:
Might be worth splitting the section into two parts (one for stage 1 and the other for stage 2).